improver_tests/acceptance/test_generate_percentiles.py

```diff
@@ -28,19 +28,19 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""Tests for the percentile CLI"""
+"""Tests for the generate-percentiles CLI"""
 
 import pytest
 
 from . import acceptance as acc
 
 pytestmark = [pytest.mark.acc, acc.skip_if_kgo_missing]
-run_cli = acc.run_cli("percentile")
+run_cli = acc.run_cli("generate-percentiles")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constant name "run_cli" doesn't conform to '(([A-Z_][A-Z0-9_]*)|(__.*__))$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5435102)